### PR TITLE
#163042447 refactoring of code

### PR DESCRIPTION
**Pull request by danwebguy**
#### What does this PR do?
A chore to refactor code and files

#### Description of Task to be completed?
Move routes, controllers, and test to a server folder.
and refactored all the code to ES6 Format

#### How should this be manually tested?
npm start
test endpoints on postman

#### What are the relevant pivotal tracker stories?
#163042447

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,26 +1,24 @@
...
 
 const app = express();
-
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 app.use(helmet());
 app.use(compression());
-
-// Use Routes
 app.use('/api/v1/users', users);
```
Comment (tosmak16) [inline server.js]: Well done @danwebguy. Please ensure to move line 17 - 20  to route folder.

File: dist/LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (tosmak16) [inline dist/LICENSE]: @danwebguy Remove dist folder from your repo. I.E add it to .gitignore file

**Review by tosmak16 — CHANGES_REQUESTED** [review]

File: server.js (hunk 1 above)
Comment (danwebguy) [reply server.js]: ok @tosmak16

File: dist/LICENSE (hunk 2 above)
Comment (danwebguy) [reply dist/LICENSE]: ok @tosmak16

Comment (danwebguy) [reply dist/LICENSE]: Done thank you @tosmak16

File: server.js (hunk 1 above)
Comment (danwebguy) [reply server.js]: Done thank you @tosmak16

**Comment by tosmak16** [comment]
@danwebguy Well done. You can go ahead and merge.

**Comment by danwebguy** [comment]
ok, thank you @tosmak16

**Pull request merged**